.github/workflows/ci.yml

```diff
@@ -85,18 +85,18 @@ jobs:
         if: contains(matrix.os, 'ubuntu')
         run: cd js && deno run -A jsr:@david/publish-on-tag@0.1.3 --allow-dirty
 
-      - name: Install cargo-llvm-cov
-        if: contains(matrix.os, 'ubuntu')
-        uses: taiki-e/install-action@cargo-llvm-cov
-
-      - name: Generate code coverage
-        if: contains(matrix.os, 'ubuntu')
-        run: cargo llvm-cov --lcov --output-path lcov.info
-
-      - name: Upload coverage to Codecov
-        if: contains(matrix.os, 'ubuntu')
-        uses: codecov/codecov-action@v3
-        with:
-          files: lcov.info
-          fail_ci_if_error: true
-          token: ${{ secrets.CODECOV_TOKEN }}
+      # - name: Install cargo-llvm-cov
+      #   if: contains(matrix.os, 'ubuntu')
+      #   uses: taiki-e/install-action@cargo-llvm-cov
+
+      # - name: Generate code coverage
+      #   if: contains(matrix.os, 'ubuntu')
+      #   run: cargo llvm-cov --lcov --output-path lcov.info
+
+      # - name: Upload coverage to Codecov
+      #   if: contains(matrix.os, 'ubuntu')
+      #   uses: codecov/codecov-action@v3
+      #   with:
+      #     files: lcov.info
+      #     fail_ci_if_error: true
+      #     token: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
FYI @crowlKats -- going to disable this for a bit.

```
[2024-12-27T22:25:11.752Z] ['error'] There was an error running the uploader: Error uploading to [https://codecov.io:](https://codecov.io/) Error: There was an error fetching the storage URL during POST: 429 - {"message":"Rate limit reached. Please upload with the Codecov repository upload token to resolve issue. Expected time to availability: 1430s."}
```

This also takes a long time to run. There must be a binary version of `cargo-llvm-cov` somewhere?

Reply:
I think maybe this is only supposed to upload and generate on main?